src/scenes/home/profile/profile.js

```diff
@@ -56,10 +56,12 @@ const Profile = ({ verified }) => (
               Get Verified for Added Benefits
             </span>
             <p>
-              In order to take advantage of resources such as Mentorship and Scholarships, we
-              require that you verify your military affiliation status. We use Id.Me for
-              verification, a highly secure and specialized platform that will protect your
-              information. We will not store or transfer any of your sensitive information.
```

Comment:
I am hesitant about removing this (We will not store or transfer any of your sensitive information) - what do the rest of you think?

Reply:
@hollomancer - I hear what you're saying.  I was thinking that would be covered by, and accounted for, on the ID.me site.  No?

If we decide to keep it, perhaps we can add that disclaimer **below** the red button.  This way, we have a clear, uncluttered path to signing up.  

Thoughts?

Reply:
I can live with that.